cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -361,8 +361,8 @@ runTxBuildCmd
   scripts <- firstExceptT ShelleyTxCmdScriptFileError $
                      mapM (readFileScriptInAnyLang . unScriptFile) scriptFiles
   txAuxScripts <- hoistEither $ first ShelleyTxCmdAuxScriptsValidationError $ validateTxAuxScripts cEra scripts
-  mpparams <- forM mProtocolParamsFile $ \ppf ->
-    firstExceptT ShelleyTxCmdProtocolParamsError (readProtocolParameters ppf)
+  mpparams <- forM mProtocolParamsSource $ \pps ->
```

Comment:
This has already been deprecated and we already query the node for the `ProtocolParameters`: https://github.com/input-output-hk/cardano-node/blob/master/cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs#L651